### PR TITLE
FEATURE: CPS-269 Add tests for c3hds

### PR DESCRIPTION
**Pull request by ondrejcernos**

### CODE REVIEW

**Comment by yenkins-admin** [comment]
Test FAILed.
Refer to this link for build results: https://ci.intgdc.com/job/cmdline-tool-tests-pull-request-builder/8/

**Comment by ondrejcernos** [comment]
co to je za fasistu?

**Comment by yenkins-admin** [comment]
Test FAILed.
Refer to this link for build results: https://ci.intgdc.com/job/cmdline-tool-tests-pull-request-builder/9/

**Comment by yenkins-admin** [comment]
Test FAILed.
Refer to this link for build results: https://ci.intgdc.com/job/cmdline-tool-tests-pull-request-builder/10/

**Comment by yenkins-admin** [comment]
Test FAILed.
Refer to this link for build results: https://ci.intgdc.com/job/cmdline-tool-tests-pull-request-builder/11/

**Comment by yenkins-admin** [comment]
Test PASSed.
Refer to this link for build results: https://ci.intgdc.com/job/cmdline-tool-tests-pull-request-builder/12/

**Comment by yenkins-admin** [comment]
Test PASSed.
Refer to this link for build results: https://ci.intgdc.com/job/cmdline-tool-tests-pull-request-builder/13/

**Comment by ondrejcernos** [comment]
@am-gd-cz please review and merge if possible

File: spec/cps/c3hds_spec.rb [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+describe 'c3hds utility' do
+  include ProjectUserManagement
+  include HDSClusterManagement
+  let(:c3hds) { '/usr/bin/c3hds' }
+
+  before(:all) do
+    @remote = Remote.by_role('c3')
+    @pgroup = create_project_group(@remote)
+    @hds_cluster_id = create_hds_cluster(@remote, 'localhost', 666,
```
Comment (am-gd-cz) [inline spec/cps/c3hds_spec.rb]: let's generate something more random

**Comment by am-gd-cz** [comment]
https://ci.intgdc.com/view/Indigo/job/cmdline-tool-tests-developer/8/ looks ok, so let's just generate more random name (in case someone will run the tests at the same time on testing cluster, e.g. checklist + someone else)

**Comment by yenkins-admin** [comment]
Test FAILed.
Refer to this link for build results: https://ci.intgdc.com/job/cmdline-tool-tests-pull-request-builder/14/

**Comment by yenkins-admin** [comment]
Test PASSed.
Refer to this link for build results: https://ci.intgdc.com/job/cmdline-tool-tests-pull-request-builder/15/

**Pull request merged**